### PR TITLE
feat(i18n): Update engine settings i18n

### PR DESCRIPTION
**Pull request by rzats**
### Contains

A few strings in the core engine are currently untranslated - these are mostly found in the new autoconfig, as well as other settings. This fixes them to be properly i18n-able!

### How to test

Browse around the settings menu to see if everything looks OK.

### Outstanding before merging

None!

### CODE REVIEW

**Comment by Cervator** [comment]
@rzats ! Now that's a name it is nice to see again. Welcome back 👋

**Comment by rzats** [comment]
Thanks @Cervator! (I still do a little lurking on Discord every now and then 👀)

File: engine/src/main/resources/org/terasology/engine/assets/i18n/menu_en.lang [hunk 1]
```diff
@@ -341,6 +346,13 @@
     "server-name": "Name",
     "server-owner": "Owner",
     "server-port": "Port",
+    "settings-chunk-timeout": "Chunk generation fail timeout (ms)",
+    "settings-chunks-till-save": "Max unloaded chunks percentage till save",
+    "settings-debug-mode": "Debug mode",
+    "settings-language": "Language",
```
Comment (DarkWeird) [inline engine/src/main/resources/org/terasology/engine/assets/i18n/menu_en.lang:352]: Duplicate entry. 
see Line 356

**Review by DarkWeird — CHANGES_REQUESTED** [review]
This PR break English Language.
```
10:23:18.705 [main] ERROR o.t.gestalt.assets.AssetType - Failed to load asset 'engine:menu_en'
java.io.IOException: Could not parse file 'menu_en.lang'
	at org.terasology.engine.i18n.assets.TranslationFormat.load(TranslationFormat.java:100)
	at org.terasology.engine.i18n.assets.TranslationFormat.load(TranslationFormat.java:38)
	....
Caused by: com.google.gson.JsonSyntaxException: duplicate key: settings-language
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:190)
	...
```
funny fact:  breaking entry - `Language` 

Ukrainian works in described place.

File: engine/src/main/resources/org/terasology/engine/assets/i18n/menu_en.lang (hunk 1 above)
Comment (DarkWeird) [reply engine/src/main/resources/org/terasology/engine/assets/i18n/menu_en.lang:352]: @rzats

Comment (rzats) [reply engine/src/main/resources/org/terasology/engine/assets/i18n/menu_en.lang:352]: Fixed this and the merge conflict!

**Review by DarkWeird — APPROVED** [review]

**Pull request merged**